File: ProjectsManager/Models/Assignment.cs
namespace ProjectsManager.Models
{
    public class Assignment
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public Project Project { get; set; }
        public Status Status { get; set; } = Status.todo;
        public User User { get; set; }
        public DateTime DueDate { get; set; } = DateTime.Now;
        public DateTime CreatedAt { get; set; } = DateTime.Now;
        public DateTime UpdatedAt { get; set; } = DateTime.Now; 
    }
}

File: AngularApp/src/app/models/assignment.ts
export class Assignment {
  createdAt: string;
  description: string;
  dueDate: string;
  id: string;
  projectId: string;
  status: string;
  title: string;
  updatedAt: string;
  userId: string;

  constructor(
    createdAt: string,
    description: string,
    dueDate: string,
    id: string,
    projectId: string,
    status: string,
    title: string,
    updatedAt: string,
    userId: string
  ) {
    this.createdAt = createdAt;
    this.description = description;
    this.dueDate = dueDate;
    this.id = id;
    this.projectId = projectId;
    this.status = status;
    this.title = title;
    this.updatedAt = updatedAt;
    this.userId = userId;
  }
}
